### ISSUE TITLE
Book 05: Sentence 012

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#3]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/2df1a29d-2381-4f17-9ae8-927e758d9d92)

```
# sent_id = Book_05_-_The_Omens.txt:12
# text = Ivin, in qujchioaia, motetzaujanj, motetzauj. etc.
# text[norm] = Ihuin, in quichihuaya, motetzahuiani, motetzahui. etc.
# text[orig] = Ivin·,·inqujchioaia·,·mo¶tetzaujanj·,·motetzauj·¬§·etc.
# text[osp] = Desta manera hazian: los que oyan el aguero, arriba dicho.
# references = A&D/161:3
1	Ivin	ihuin	ADV	_		4	_	_	Folio=2r|Paragraph=1|Line=14|Norm=Ihuin|Analysed=Yes
2	,	,	PUNCT	_		4	punct	_	Folio=2r|Paragraph=1|Line=14|Norm=,|Analysed=Yes
3	in	in	DET|SCONJ	_	_	4	_	_	Orig=inqujchioaia|Folio=2r|Paragraph=1|Line=14|Norm=in
4	qujchioaia	chihua	VERB	_	_	0	root	_	Orig=inqujchioaia|Folio=2r|Paragraph=1|Line=14|Norm=quichihuaya|Ambiguous=Yes|Analysed=Yes|Guessed=Yes
5	,	,	PUNCT	_		6	punct	_	Folio=2r|Paragraph=1|Line=14|Norm=,|Analysed=Yes
6	motetzaujanj	tetzahuia	VERB	_	_	4	_	_	Orig=mo¶·tetzaujanj|Folio=2r,2r|Paragraph=1,1|Line=14,15|Norm=motetzahuiani
7	,	,	PUNCT	_		8	punct	_	Folio=2r|Paragraph=1|Line=15|Norm=,|Analysed=Yes
8	motetzauj	tetzahuia	VERB	_	_	6	_	_	Folio=2r|Paragraph=1|Line=15|Norm=motetzahui
9	.	.	PUNCT	_		4	punct	_	Folio=2r|Paragraph=1|Line=15|Norm=.|Analysed=Yes
10	etc.	etc.	NOUN	_	State=Abs	4	_	_	Folio=2r|Paragraph=1|Line=15|Norm=etc.|Analysed=Yes
```

**Comment by Lguyogiro** [comment]
- Here, I have a suspicion motetzahuiani is NOUN, "he/she who is affected by portents."
- `in` should be SCONJ, and `mark(quichihuaya, in)`
- `nsubj(quichihuaya, motetzahuiani)`
- not sure about the rel between motetzahuiani and motetzauh, I want to say `appos` but it is strange since one would be noun and the other verb...but it looks like that.

**Comment by ftyers** [comment]
> Here, I have a suspicion motetzahuiani is NOUN, "he/she who is affected by portents."

Do we want this as a `NOUN` or as a `VERB` with `VerbForm=Vnoun`? The fact that it has some internal agreement is suggestive...

> in should be SCONJ, and mark(quichihuaya, in)

What is it subordinating?

>  nsubj(quichihuaya, motetzahuiani)

Agree with this one (or `csubj`, depending on the above).

> not sure about the rel between motetzahuiani and motetzauh, I want to say appos but it is strange since one would be noun and the other verb...but it looks like that.

I wonder if this is actually *motetzahui* (note the *-i* on the end), López Austín has *De esta manera lo hacía el que tenía el agüero, el que había te­nido augurio, etcétera.* I feel like they should both be `NOUN` or both be `VERB`. 

> in the EWT treebank, etc. consistently has the conj rel

Yeah, although it's weird also to `conj` a `NOUN` and a `VERB`.

**Comment by Lguyogiro** [comment]
> Do we want this as a NOUN or as a VERB with VerbForm=Vnoun? The fact that it has some internal agreement is suggestive...

Not sure, is Vnoun used in other treebanks for deverbal agentives?  I wonder if you could have specific object agreement in this nominalized form, like "quimotzahuiani" or "nechmotzahuiani".

**Comment by ftyers** [comment]
I tried to find examples of `[^ ]+mo[^ ]+ni`, there are not many, but there is:
> In tetzon, ocotl tlahuilli quitocani, tezcatl itech mixcuicuitiani, tlilli tlapalli, quitemoani quitocani (Book 10)

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/5fd41df8-59cc-40cb-ab2a-2936d07b675c)

Although not sure if this is the same *-ni*.

**Comment by Lguyogiro** [comment]
They look the same to me, especially given the translations. 

My inclination is towards treating them as derived nouns (NOUN not VERB). The UD documentation on Vnoun seems to suggest it for infinitives.

**Comment by ftyers** [comment]
Ok, if we make them both nouns, then the question is what is *in quichihuaya* subordinating. Is it something like:
> Así es que hacía el que oía el agüero, el que tenía augurio, etc.

**Comment by Lguyogiro** [comment]
what if _ivin_ is `root`. e.g. "fue de esta manera que hacían...". This would be similar to the clefting we have in the nhi treebank.

**Comment by ftyers** [comment]
Yeah, I was wondering that, so
```
root(Ihuin)
csubj(Ihuin, quichihuaya)
mark(quichihuaya, in)
```
What do you think?

**Comment by Lguyogiro** [comment]
Is this one good to go?

**Comment by ftyers** [comment]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/58aacbe1-9d53-4e82-8471-4a70cdfa2aef)

Like this?

**Comment by ftyers** [comment]
Fixed in c45e9ef.
